### PR TITLE
admin service tutorials

### PR DESCRIPTION
**Pull request by mishaschwartz**

### CODE REVIEW

**Review by alexandercyu — APPROVED** [review]

File: tutorials/admin/services/services.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+tools that largely fall under the following categories:
+
+- serving and cataloging data hosted on your server
+- interactive development environment
+- remote processing tools
+
+(enable)=
+## Enable an included service
+
+You can pick and choose which services you would like to offer your users by enabling them via the `env.local` file.
```
Comment (dchandan) [inline tutorials/admin/services/services.md:13]: Let's preface this sentence with "The Birdhouse stack comes loaded with several optional services." Or something to that effect.

Comment (dchandan) [reply tutorials/admin/services/services.md:13]: Actually, you see to have something like that in the next paragraph. I'd make that as the leading paragraph of this section and then add move this sentence to the end of that paragraph.

File: tutorials/admin/services/services.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+
+The Marble software comes with several services already included. The list of services already included can be found
+in the 
+[Birdhouse documentation](https://birdhouse-deploy.readthedocs.io/en/latest/birdhouse/components/README.html).
+
+This documentation explains what each service is used for and how to enable and configure it.
```
Comment (dchandan) [inline tutorials/admin/services/services.md:19]: You mean the document linked above in the preceding paragraph? If so, then I think this remark shouldn't be on a separate paragraph, but instead should be incorporated within the other paragraph.

File: tutorials/admin/services/services.md [hunk 3]
```diff
@@ -1 +1,75 @@
...
+
+In general, the process to enable a service is a follows:
+
+1. Take note of the directory that contains the service's code. In general this will be on of the directories in the
```
Comment (dchandan) [inline tutorials/admin/services/services.md:23]: "one of the"

File: tutorials/admin/services/services.md [hunk 4]
```diff
@@ -1 +1,75 @@
...
+   for customization options for each service).
+4. Restart the software by running:
+
+   ```shell
+   ./pavics-compose.sh down
+   ./pavice-compose.sh up -d
+   ```
+
+   in order for the changes to take effect.
+5. Log in to Magpie as an admin user to [set the access permissions for the new service](../users/permissions.md)
```
Comment (dchandan) [inline tutorials/admin/services/services.md:39]: Let's link to Magpie documentation. By the way, where would Magpie be introduced in the admin side tutorials? Are we planning on writing a brief introduction somewhere about what Magpie does within Birdhouse?

**Review by dchandan — CHANGES_REQUESTED** [review]

File: tutorials/admin/services/services.md [hunk 5]
```diff
@@ -1 +1,75 @@
-# services
+# Services
+
+The Marble software can be configured to provide several services to the users of your node. Services are user-facing
+tools that largely fall under the following categories:
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply tutorials/admin/services/services.md:4]: Does "largely" not imply that these categories are not all-inclusive? I could replace with "generally", "mostly", "mainly", ...

File: tutorials/admin/services/services.md (hunk 4 above)
Comment (mishaschwartz) [reply tutorials/admin/services/services.md:39]: Yeah, it's introduced in that file that's linked (admin/users/permissions.md). See #33

File: tutorials/admin/services/services.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dchandan) [reply tutorials/admin/services/services.md]: How about: "Services are user-facing tools and the Birdhouse stack comes with several optional services that largely fall under the following categories:"?

My thinking here is that this should clarify that the three categories listed below apply specifically to the services that come included in Birdhouse.

**Review by dchandan — APPROVED** [review]

**Comment by dchandan** [comment]
I think this is ready to merge.

**Pull request merged**